xml/dm_registers.xml

```diff
@@ -429,6 +429,14 @@
     <register name="Device Tree Addr 2" short="devtreeaddr2" address="0x1b"/>
     <register name="Device Tree Addr 3" short="devtreeaddr3" address="0x1c"/>
 
+    <register name="Next Debug Module" short="nextdm" address="0x1d">
+        If there is more than one DM accessible on this DMI, this register
```

Comment:
oh, for some reason I assumed these would be chained at the DTM level, not the DebugModule level. I guess this works too and is more efficient than JTAG chaining.

Reply:
So this suddenly means that all these addresses are actually offsets from some base, and the first one happens to be 0? I don't know if you made that clear.

Reply:
More DMs can be added at the DTM level also, but this seems like a trivial extension that can be more efficient, and is also DTM-agnostic.

I'll some blurb about a base address somewhere.